Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -6637,6 +7738,318 @@ def certificate_intermediate_group_list_command(
     return command_results
 
 
+def url_access_rule_group_create_command(
+    client: Client, args: dict[str, Any]
+) -> CommandResults:
+    """Create an URL access rule group.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    client.validate_url_access_group(args=args)
+    name = args["name"]
+    response = client.url_access_rule_group_create_request(
+        name=name,
+        action=args["action"],
+        trigger_policy=args.get("trigger_policy"),
+        severity=args.get("severity"),
+        host_status=args["host_status"],
+        host=args.get("host"),
+    )
+    command_results = generate_simple_command_results(
+        object_type=OutputTitle.URL_ACCESS_RULE_GROUP.value,
+        object_name=name,
+        action=OutputTitle.CREATED.value,
+        response=response,
+    )
```

Comment:
Dont we want to add anything to the context data? It seems that the generate_simple_command_results returns CommandResutls only with readable outputs and raw response